### ISSUE TITLE
KITTI test metric?

### ISSUE THREAD

**Issue by csuwoshikunge** [YvanYin/VNL_Monocular_Depth_Prediction#26]
Thanks for your sharing, I have two questions on "how to test the error metric on KITTI?"  As said in the paper, you use the Eigen split. But KITTI provides an offical train/validation split, 
1) what's the difference between these two splits? It seems that Eigen used the raw depth map which is sparser than the official train/validation depth map?
 2) the depth ground truth (KITTI provides) is very sparse, do you test the error metric on these sparse gt or inpainted dense gt?
Thanks for any help.

**Comment by YvanYin** [comment]
Hi,
1. Eigen split the train/val of official parts to train/val/test.

2. I test on GT sparse depth instead of dense gt. However, I train the model on inpainted dense gt. Therefore, if the model is finetuned on GT sparse depth map, the performance maybe can improve.
